pkg/cryptoauthlib/Makefile.include

```diff
@@ -0,0 +1,16 @@
+PKG_BUILDDIR ?= $(PKGDIRBASE)/cryptoauthlib
```

Comment:
`Makefile.include` is globally included, thus it cannot use `PKG_BUILDDIR`. This maybe works if cryptoauthlib is the only pkg doing that, but *will* cause problems in the future.
Maybe namespace (rename to e.g., `CRYPTOAUTHLIB_BUILDDIR` or something shorter but unique?

There's a similar issue with `TESTINCLDIR`. It should be namespaced if used in the global `Makefile.include` namespace.